18F/identity-idp
# FormSteps: Add support for asynchronous submission

**Why**: Since some steps involve a call and response from an API before continuing to the next step.

**Testing Instructions:**

Since we don't currently implement this anywhere, you can simulate a delayed submission on the personal key step by locally reverting test code removed in 3a6596c.

1. `git revert 3a6596c`
2. Ensure `idv_api_enabled: true` in local `config/application.yml`
3. Sign in and proof a user up to the personal key step
4. Click "Continue"
5. Observe spinner button spins for 2 seconds prior to showing confirmation modal.
